bindings/python/meson.build

```diff
@@ -0,0 +1,26 @@
+project(
+  'tree-sitter-capnp',
+  'c',
+  license: 'MIT'
+)
+
+python = import('python').find_installation('python3', modules: ['cffi'], pure: false)
+python_dep = python.dependency(required: true)
+
+ts_cffi_srcs = configure_file(
+  command: [python, files('ffibuilder.py'), '@OUTPUT@'],
+  output: '_tree_sitter_capnp.c',
+)
+
+ts_pyext = python.extension_module(
+    '_tree_sitter_capnp',
+    ts_cffi_srcs,
+    files('..' / '..' / 'src' / 'parser.c'),
```

Comment:
(files objects aren't strictly necessary here, again, since it's a parameter to a function that explicitly takes files as either string or files())

Note that meson doesn't require you to use the `/` path-joining operator for statically coded paths like this, internally it just operates on unix-style forward-slash paths. The path-joining operator is mainly *useful* for handling cases where you add together paths that you don't control, which may be absolute and may or may not have trailing "/", which causes string `+` concatenation to potentially cause bad things.

Here you just have one path segment: `'../../src/parser.c'`.